core/tabs.go

```diff
@@ -20,6 +20,33 @@ import (
 	"cogentcore.org/core/tree"
 )
 
+// Tabber is an interface for getting the parent Tabs of tab buttons.
+// It exposes the main Tabs interface so other packages can build on that
+// to provide an augmented Tabs API.
+type Tabber interface {
```

Comment:
What is this purpose of this? We have decided that these kinds of large, duplicative interfaces are not good.

Reply:
This is essential for the "Lab" (was databrowser) api.

Reply:
You have `lab.Tabber` that extends `core.Tabber` and has a bunch more methods!? There is no way that all of these methods need to be extended and handled in all of these cases; use `OnShow` or some other dynamic extension to accomplish your requisite functionality; you are not going to pollute the basic API with this massive interface.

Reply:
`AsCoreTabs` is fine, but you are not supposed to have duplicate methods in the interface.

Reply:
ok minimal now.